lib/thunder/pages/thunder_page.dart

```diff
@@ -399,191 +403,198 @@ class _ThunderState extends State<Thunder> {
       ],
       child: WillPopScope(
         onWillPop: () async => _handleBackButtonPress(scaffoldMessengerKey.currentState),
-        child: BlocListener<DeepLinksCubit, DeepLinksState>(
```

Comment:
Here you can switch to a  [MultiBlocListener](https://pub.dev/documentation/flutter_bloc/latest/flutter_bloc/MultiBlocListener-class.html)

Reply:
Good idea! Done!